pkg/scheduler/metrics/metrics.go

```diff
@@ -87,6 +87,38 @@ var (
 			Name:      "total_preemption_attempts",
 			Help:      "Total preemption attempts in the cluster till now",
 		})
+	AssumeVolumesLatency = prometheus.NewHistogram(
+		prometheus.HistogramOpts{
+			Subsystem: schedulerSubsystem,
+			Name:      "assume_bind_volumes_latency_microseconds",
+			Help:      "Assume and bind volumes latency",
+			Buckets:   prometheus.ExponentialBuckets(1000, 2, 15),
+		},
+	)
+	VolumeBindingPredicateLatency = prometheus.NewHistogram(
+		prometheus.HistogramOpts{
+			Subsystem: schedulerSubsystem,
+			Name:      "volume_binding_predicate_latency_microseconds",
+			Help:      "Volume binding predicate latency",
+			Buckets:   prometheus.ExponentialBuckets(1000, 2, 15),
+		},
+	)
+	BindPodVolumeLatency = prometheus.NewHistogram(
+		prometheus.HistogramOpts{
+			Subsystem: schedulerSubsystem,
+			Name:      "bind_pod_volume_latency_microseconds",
+			Help:      "Bind pod volume latency",
+			Buckets:   prometheus.ExponentialBuckets(1000, 2, 15),
+		},
+	)
+
+	VolumeBindingFailed = prometheus.NewCounter(
+		prometheus.CounterOpts{
+			Subsystem: schedulerSubsystem,
+			Name:      "total_volume_binding_failed",
```

Comment:
I think the convention is to have count or latency at the end of the name, so something like "volume_binding_error_count"